tests/test_setup_prometheus_adapter_config.py

```diff
@@ -21,15 +21,6 @@
 @pytest.mark.parametrize(
     "autoscaling_config,expected",
     [
-        (
-            {
-                "metrics_provider": "mesos_cpu",
-                "decision_policy": "bespoke",
-                "moving_average_window_seconds": 123,
-                "setpoint": 0.653,
-            },
-            False,
-        ),
```

Comment:
I see for the next test cases you're deleting, there are equivalents using `cpu` instead of `mesos_cpu`, but I don't see one here.  Is this alright that we no longer have coverage for cpu-autoscaling with this set of options?

Reply:
Good catch @jfongatyelp ! I have added the missing test case.